js/background.js

```diff
@@ -28,8 +28,11 @@ function Background() {
 
   localStorage['os'] = os;
 
-  if (localStorage['prev_version'] === undefined) {
-    localStorage['prev_version'] = chrome.runtime.getManifest().version;
+  chrome.runtime.onInstalled.addListener(function(details) {
```

Comment:
@bsstoner I managed to find a way of triggering this only on install. Here is the relevant part of the docs: https://developer.chrome.com/extensions/runtime#event-onInstalled